__tests__/fixtures/action_usage_readme.output

```diff
@@ -0,0 +1,21 @@
+<!-- action-docs-usage project="npalm/action-docs" version="v1" -->
+## Usage
+
+```yaml
+- uses: npalm/action-docs@v1
+  with:
+    inputA:
```

Comment:
An alternative approach could be to print the default value between < > behind the parameter and in comment after that the description. But that would be a bit complicated when the description is multi-line.

Reply:
If I'm reading you correctly, you're saying if the default here for inputA is 0 it would look like this:

```yaml
- uses: npalm/action-docs@v1
  with:
    inputA: 0
    #
    # Here would be the other stuff that appears in the usage guide
```

I think my objection to that would be that if there's a default value and you're not overriding it, you should be able to safely just omit the parameter. If we include the parameter then are we encouraging users to specify values even when they're just the default?

Reply:
Oh I guess another part of this suggestion is to do something more like this

```yaml
...
    inputA: 0 # the description would be here
...

But I agree it might get a bit cluttered?
```

Reply:
But this approach required to remove line breaks in the description. For me it is fine if to accept the PR as is. And approve later

Reply:
The idea was more like this

...
    inputA: <your valiue> # the description would be here
...

or 

...
    inputA: <default value> # the description would be here
...

But in both cases with < > to suggest to change by the user